src/packages.builds

```diff
@@ -33,6 +39,21 @@
 
   <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipNativePackageBuild)' != 'true'" >   
     <!-- add specific native builds / pkgproj's here to include in servicing builds -->
+    <Project Include="Native\pkg\**\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
```

Comment:
Are we really shipping all native packages? 

No need to duplicate the SkipNativePackageBuild condition as it is on the itemgroup.

Reply:
Base on your other comment you are going to need to rebuild all and not just the new distros.

Reply:
We need to ship all of the native packages for each new distro, which requires updating *targets, at which point it makes sense to also update the runtime packages that aren't changing so that we can stick to one *targets version.

I pushed an update that causes all native packages to be rebuilt for all OSGroups